.gitignore

```diff
@@ -1,7 +1,11 @@
-*.gem
-*.swp
-.rvmrc
-/.idea/
+# Ignore the RubyMine configuration files.
+.idea/
```

Comment:
Would you mind adding `.vscode` for those of us living in the future? 😉

Reply:
Done